lighthouse-core/scripts/generate-bundlephobia-database.js

```diff
@@ -0,0 +1,178 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+'use strict';
+
+/* eslint-disable no-console */
+
+/** @typedef {import('bundle-phobia-cli').BundlePhobiaLibrary} BundlePhobiaLibrary */
+
+const fs = require('fs');
+const path = require('path');
+const getPackageVersionList = require('bundle-phobia-cli').fetchPackageStats.getPackageVersionList;
+const fetchPackageStats = require('bundle-phobia-cli').fetchPackageStats.fetchPackageStats;
+const databasePath = path.join(__dirname,
+  '../lib/large-javascript-libraries/bundlephobia-database.json');
+
+/** @type {Record<string, string[]>} */
+const suggestionsJSON = require('../lib/large-javascript-libraries/library-suggestions.js')
+  .suggestions;
+
+
+/** @type {string[]} */
+let largeLibraries = [];
+largeLibraries = largeLibraries.concat(...Object.keys(suggestionsJSON));
+
+/** @type {string[]} */
+let suggestedLibraries = [];
+suggestedLibraries = suggestedLibraries.concat(...Object.values(suggestionsJSON));
```

Comment:
same, but with flattening:
```suggestion
const suggestedLibraries = Object.values(suggestionsJSON).reduce((arr, lib) => arr.concat(lib), []);
```